test/serdes_test.c

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <string.h>
+#include <openssl/evp.h>
+#include <openssl/pem.h>
+#include <openssl/rsa.h>
+#include <openssl/x509.h>
+#include <openssl/asn1.h>
+#include <openssl/serializer.h>
+#include <openssl/deserializer.h>
+
+#include "testutil.h"
+
+static EVP_PKEY *key_RSA = NULL;
+
+static EVP_PKEY *make_RSA(int rsa_type)
+{
+    EVP_PKEY *pkey = NULL;
+    EVP_PKEY_CTX *ctx = EVP_PKEY_CTX_new_id(rsa_type, NULL);
```

Comment:
Use the API that doesnt automatically set up a legacy pmeth table.
EVP_PKEY_CTX *ctx = EVP_PKEY_CTX_new_from_name(NULL, "RSA", NULL);

Reply:
Done